test/EFCore.SqlServer.FunctionalTests/Query/PrimitiveCollectionsQueryOldSqlServerTest.cs

```
@@ -453,6 +453,21 @@ SELECT COUNT(*)
 """);
     }
 
+    public override async Task Inline_collection_Contains_with_EF_Parameter(bool async)
+    {
+        await base.Inline_collection_Contains_with_EF_Parameter(async);
```

This one is interesting... I think we should actually throw an error if the user uses EF.Parameter() and the SQL compatibility is too low. After all, the user has very clearly indicated their intent with EF.Parameter(), but it's not possible to fulfil that intent...

What do you think?

---

Fine with me.